templates/controllers/grid/settings/genre/form/genreForm.tpl

```diff
@@ -31,6 +31,9 @@
 {fbvFormSection title="manager.setup.groupType" for="category"}
 	{fbvElement type="select" id="category" from=$submissionFileCategories selected=$category translate=false}
 {/fbvFormSection}
+{fbvFormSection title="manager.setup.genres.key" for="key"}
+	{fbvElement type="text" id="key" value=$key maxlength="30" readonly=$readOnly size=$fbvStyles.size.SMALL}
```

Comment:
I think it might be necessary to add a note about this, since it's not at all evident what it's for. Something like: "An optional short symbolic identifer for this genre."